<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes app</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="lstNotes">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>551</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNotes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnArchive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Archive</string>
    </property>
   </widget>
   <widget class="QListView" name="lstArchive">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>551</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="wgtNewnote">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>641</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QTextEdit" name="ttlEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>491</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>491</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save note</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnArch">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[↓]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSavechng">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUnarch">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[↑]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancelarch">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSavechngArch">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDeletearch">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QListView" name="lstLists">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>240</y>
       <width>121</width>
       <height>192</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblLists">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>210</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lists</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNew">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New +</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblList">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
